pyomo/contrib/parmest/parmest.py

```diff
@@ -234,6 +234,21 @@ def SSE(model):
     expr = sum((y - y_hat) ** 2 for y, y_hat in model.experiment_outputs.items())
     return expr
 
+def regularize_term(model, prior_FIM, theta_ref):
+    """
+    Regularization term for the objective function, which is used to penalize deviation from a 
+    reference theta
+    (theta - theta_ref).transpose() * prior_FIM * (theta - theta_ref)
+
+    theta_ref: Reference parameter value, element of matrix
+    prior_FIM: Fisher Information Matrix from prior experimental design, matrix
+    theta: Parameter value, matrix
+
+    Added to SSE objective function
+    """
+    expr = ((theta - theta_ref).transpose() * prior_FIM * (theta - theta_ref) for theta in model.unknown_parameters.items())
```

Comment:
Does this run? My intuition is that you need to write out the matrix multiplication and cannot use matrix multiplication.